js/src/components/google-mc-account-card/google-mc-account-card.js

```diff
@@ -23,7 +23,12 @@ const GoogleMCAccountCard = () => {
 		return <NonConnected />;
 	}
 
-	return <ConnectedGoogleMCAccountCard googleMCAccount={ googleMCAccount } />;
+	return (
+		<ConnectedGoogleMCAccountCard
+			hideNotificationService
```

Comment:
It seems like `hideNotificationService` isn't defined. I believe its value will always be `false`. Is this intentional, or am I missing something?

Reply:
This was introduced in my PR. In the settings is false, in Onboarding is true @jorgemd24

Reply:
Yeah, the prop with just its name with any value is considered true.